src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="form"
+      action="/"
+      method="post"
+      data-qa="big"
+      >
+      <input
+        type="search"
+        name="big__search"
+        id="big-search"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+        class="input input__big"
+      >
+      <img
+        class="searchicon searchicon__big"
+        src="./images/Search.svg"
+        alt="search"
+      >
+    </form>
+    <form
+      class="form"
+      action="/"
+      method="post"
+      data-qa="small"
+      >
+      <input
+        type="search"
+        name="small__search"
+        id="small-search"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+        class="input input__small"
+      >
+      <img
+        class="searchicon searchicon__small"
+        src="./images/Search.svg"
+        alt="search"
+      >
+    </form>
   </body>
-</html>
+</html>
```

Comment:
![image](https://user-images.githubusercontent.com/99331793/197971577-51532398-f546-49ed-bd2c-4233d67cc080.png)

add blank line in the end of the file